---
layout: cv
title: Koen Rademaker's CV
---
# Koen Rademaker

MSc student Bioinformatics and Systems Biology, Vrije Universiteit Amsterdam / Universiteit van Amsterdam.<br/>

<a href="k.j.rademaker@student.vu.nl">k.j.rademaker@student.vu.nl</a> 

<div id="webaddress">
  <a href="http://orcid.org/0000-0003-3059-5183"><i class="ai ai-orcid"></i> 0000-0003-3059-5183</a> -
  <a href="https://github.com/krademaker"><i class="ai github"></i> GitHub</a> - 
  <a href="https://twitter.com/kjrademaker"><i class="ai ai-twitter"></i> @kjrademaker</a> -
  <a href="https://www.linkedin.com/in/koen-rademaker"><i class="ai linkedin"></i> LinkedIn</a>
</div>

<!-- ### Qualities Many. -->

<!-- ### Interests Computational cancer biology, single cell data, (quasi) mechanistic modeling. -->

<!-- ### Research questions Que? -->


## Experience

`Apr 2023 - Sep 2023`
**MSc internship**, *Computational Genomics and System Genetics group (Oliver Stegle), German Cancer Research Center (Heidelberg)*.<br>
Statistical modelling of cell type-specific expression changes in neurodevelopmental disorders from 10x Genomics Visium spatial transcriptomics data. Supervised by prof. Dr. Oliver Stegle and Dr. Alexander Aivazidis (Wellcome Sanger Institute) in collaboration with Bayraktar group (Sanger).


`May 2022 - Jun 2022` **Teaching Assistant**, *University of Amsterdam*.<br>
* Image Processing and Quantitative Data Analysis (Dr. Marten Postma)


`Sep 2021 - Feb 2022`
**MSc internship**, *Chromatin Genomics group (Bas van Steensel), Netherlands Cancer Institute (Amsterdam)*.<br>
Developed pipeline for mapping genomic rearrangement experiments and downstream analyses. Supervised by prof. Dr. Bas van Steensel, Dr. Lise Dauban, Mathias Eder MSc.


`Jan 2021 - May 2022` **Teaching Assistant**, *Vrije Universiteit Amsterdam*.<br>
* Structural Bioinformatics (2021)
* Bioinformatics for Translational Medicine (2021, 2022)
* Introduction to Bioinformatics (2022)


`Sep 2020 - Jun 2022` **Peer Mentor**, *Vrije Universiteit Amsterdam*.<br>
I helped groups of new students (2020/2021 and 2021/2022 cohorts) to navigate their way around the MSc program in Bioinformatics & Systems Biology through regular informal meetings.


`Feb 2019 - Sep 2019`
**BSc internship**, *Department of Translational Neuroscience, UMC Utrecht*.<br>
Identification of neuronal cells involved in schizophrenia through integration of scRNA-seq and GWAS data for cell type enrichment methods DEPICT, MAGMA, LDSC and FUMA. Supervision by Dr. Bochao Lin.


`Sep 2017 - Jan 2018`
**BSc internship**, *Translational Metabolic Laboratory, Radboudumc (Nijmegen) *.<br>
Application of bioinformatics for glycopeptide profiling to study congenital disorders of glycosylation. Supervised by Dr. Hans Wessels.


## Education

`2019-2023`
**MSc Bioinformatics and Systems Biology**, *Vrije Universiteit Amsterdam / University of Amsterdam*<br/>
Thesis: "Mapping cell type-specific differential expression of trisomy 21 in spatially resolved fetal brain development."<br/>
Supervisor: Prof. Dr. Oliver Stegle (German Cancer Research Center), Dr. Alexander Aivazidis (Wellcome Sanger Institute).

`2015-2019`
**BSc Bioinformatics**, *HAN University of Applied Sciences*.<br/>
Thesis: "Targeting brain cell types underlying schizophrenia."<br/>
Supervisor: Dr. Bochao Lin, UMC Utrecht.


<!-- ## Coding

`Proficient`
Language 1

`Intermediate`
Language 2

`Rusty`
Language 3-->

## Publications

Publications are also listed on [ORCID](https://orcid.org/0000-0003-3059-5183).

### Peer-reviewed publications

`2022`

M. Olislagers, **K. Rademaker**, R. A. H. Adan, B. D. Lin & J. J. Luykx., 'Comprehensive analyses of RNA-seq and genome-wide data point to enrichment of neuronal cell type subsets in neuropsychiatric disorders'. *Mol Psychiatry*, *27*, 947–955 (2022); doi:[10.1038/s41380-021-01324-6](https://www.doi.org/10.1038/s41380-021-01324-6).<br>

### Pre-prints

M. Olislagers, **K. Rademaker**, R. A. H. Adan, B. D. Lin & J. J. Luykx. ‘Comprehensive analyses of RNA-seq and genome-wide data point to enrichment of neuronal cell type subsets in neuropsychiatric disorders’. _bioRxiv_, doi: [10.1101/2021.05.06.442982](https://doi.org/10.1101/2021.05.06.442982 ) (July 20 2021).<br>
